metadata/item_general.py

```diff
@@ -373,13 +390,13 @@
     "Quick Change",
     "D Down Pound",
     "Dizzy Stomp",
-    "Smash Charge",
```

Comment:
Is this change intentional? (same with Jump Charge a few lines down)

Reply:
It is. The items those two lines represent are actually the beta charge badges, but were misnamed before. The regular jump and smash charge show up elsewhere on the same list.
This list reproduces the data array in the mod repo's MainMenu.patch file, which is part of the seed hash icons on the save select screen.